MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,16 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChanged: Observer<String?>?
```

Comment:
To follow the naming convention, please use 'change' instead of 'changed':

```suggestion
	var onErrorStateChange: Observer<String?>?
```